docs/laravel-integration.md

```diff
@@ -235,21 +250,27 @@ if ($zipkinTracer) {
     $span->setAttribute('foo', 'bar');
     $span->updateName('New name');
 
-    $childSpan = $zipkinTracer->startAndActivateSpan('Child span');
+    $childSpan = $zipkinTracer->spanBuilder('Child span')->startSpan();
+    $childScope = $childSpan->activate();
     try {
         throw new \Exception('Exception Example');
     } catch (\Exception $exception) {
-        $span->setSpanStatus($exception->getCode(), $exception->getMessage());
+        $childSpan->setSpanStatus($exception->getCode(), $exception->getMessage());
```

Comment:
Span::setStatus() accepts StatusCode::STATUS_* now, and this should now use Span::recordException. See SpanInterface

Reply:
Should we still set a status or only record the exception without marking the error status?